sdk/all/build.gradle

```diff
@@ -0,0 +1,54 @@
+plugins {
+    id "java"
+    id "maven-publish"
+
+    id "me.champeau.gradle.jmh"
+    id "ru.vyarus.animalsniffer"
+}
+
+description = 'OpenTelemetry SDK'
+ext.moduleName = "io.opentelemetry.sdk"
+ext.propertiesDir = "build/generated/properties/io/opentelemetry/sdk"
+
+dependencies {
+    api project(':opentelemetry-api'),
+            project(':opentelemetry-sdk-common'),
+            project(':opentelemetry-sdk-correlation-context'),
+            project(':opentelemetry-sdk-metrics'),
+            project(':opentelemetry-sdk-tracing')
```

Comment:
I am not a gradle expert, by any means, but I think we might want to actually have this module have multiple source sets, so downloading it from maven central, or wherever, is a single artifact, rather than multiple artifacts, which is what having dependent modules will result in.

Reply:
That's just for "all" which is the new "sdk" right?

I'm ok with doing that if someone can explain/show how.

Reply:
Here's an example of this being done: https://github.com/newrelic/newrelic-telemetry-sdk-java/pull/214/files

Reply:
and, yes, just for the 'all' artifact.

Reply:
I generally find multiple sources to have a higher chance of biting later (if both source files end up in the final app which is reasonably likely). Is there an advantage to the fat jar for an SDK? Don't feel that strongly one way or another but would lean towards letting Maven do it's thing, it's pretty good at it

Reply:
Can someone confirm whether or not I should make this change?

Reply:
I don't feel super strongly about it, so if someone cares a lot, we can have it as a follow-up issue/PR.